MySQLdb/connections.py

```diff
@@ -132,6 +132,11 @@ class object, used to create cursors (keyword only)
         :param bool local_infile:
             enables LOAD LOCAL INFILE; zero disables
 
+        :param bool secure_auth:
+            If False, secure_auth is disabled.
+            If True, secure_auth is enabled.
+            If None, secure_auth isn't set and server default is used.
```

Comment:
Is it really support `None`?  Maybe, "When not specified,"?

Is it really "server default"?
Isn't it a "client library default"?

And please add note about this option is deprecated.